packages/flutter/lib/src/material/popup_menu.dart

```diff
@@ -1064,6 +1094,8 @@ Future<T?> showMenu<T>({
 }) {
   assert(items.isNotEmpty);
   assert(debugCheckHasMaterialLocalizations(context));
+  assert(position != null || positionBuilder != null,
+  'Either position or positionBuilder must be provided.');
```

Comment:
I wonder if we can be more strict on it, and only allow one of them,

```
assert((position != null) != (positionBuilder != null),
```

Reply:
done